src/components/canrawsender/canrawsender_p.h

```diff
@@ -47,6 +50,22 @@ class CanRawSenderPrivate : public QObject {
         }
     }
 
+    void saveSettings(QJsonObject& json) const
```

Comment:
I would create `canrawsender_p.cpp` and move implementation there. This (+forward declarations) will limit compilation times and will make code easier to maintain.